tests/src/test/java/alluxio/client/rest/TestCase.java

```diff
@@ -85,10 +87,17 @@ public String getMethod() {
    */
   public URL createURL() throws Exception {
     String url = String.format("http://%s:%s%s/%s", mHostname, mPort, mBaseUri, mEndpoint);
-    String params = Joiner.on("&").withKeyValueSeparator("=").join(mParameters.entrySet());
-    if (params.length() > 0) {
-      url = String.format("%s?%s", url, params);
```

Comment:
I know technically `?key=` and `?key` are equivalent but since our docs are going to be using the latter notation, we should do the same in our tests.

Unfortunately, doing is is quite verbose and expensive via stream processing. Instead I could replace this with plain `Map.EntrySet` processing for-loops. I'm not sure which we'd prefer for performance/readability. @yuzhu @rongrong for input please 🙏

Reply:
Either way is fine. This is slightly less readable but pretty easy to understand.